src/views/utils.js

```diff
@@ -46,3 +49,50 @@ export function getDefaultInitialViewState(loader, viewSize, zoomBackOff = 0) {
   };
   return loaderInitialViewState;
 }
+
+/**
+ * Creates the layers for viewing an image in detail.
+ * @param {string} id The identifier of the view.
+ * @param {Object} props The layer properties.
+ * @returns {Array} An array of layers.
+ */
+export function getDetailLayers(id, props) {
```

Comment:
```suggestion
export function getImageLayers(id, props) {
```
nit, but don't like the name clash between "detail" from detail-overview and using it again for side-by-side